azurerm/internal/services/network/web_application_firewall_policy_resource.go

```diff
@@ -376,8 +400,11 @@ func resourceWebApplicationFirewallPolicyRead(d *schema.ResourceData, meta inter
 		if err := d.Set("managed_rules", flattenWebApplicationFirewallPolicyManagedRulesDefinition(webApplicationFirewallPolicyPropertiesFormat.ManagedRules)); err != nil {
 			return fmt.Errorf("Error setting `managed_rules`: %+v", err)
 		}
-		if err := d.Set("managed_rules", flattenWebApplicationFirewallPolicyManagedRulesDefinition(webApplicationFirewallPolicyPropertiesFormat.ManagedRules)); err != nil {
-			return fmt.Errorf("Error setting `managed_rules`: %+v", err)
```

Comment:
Is there a reason we are removing managed rules from here?

Reply:
It's a good question, but the answer is above, as the lines were duplicated by mistake in my original contribution of `managed_rules`:

https://github.com/terraform-providers/terraform-provider-azurerm/commit/1b5a0b36d5c3a756e2efe45a3eff2bda67135997#diff-25ef614a1fbe2de62ec91d10ddde72c778c5876a77dc676908b065a8268846c5R344-R349

So it _should_ be safe 😅

Reply:
🤦‍♀️  - that they are and i'membarrassedd that i missed that

Reply:
> 🤦‍♀️  - that they are and i'membarrassedd that i missed that

Don't be! I bet you have a lot on your plate with all the things happening in this provider. Thanks for the review! 😊